src/main/java/com/android/volley/toolbox/DiskBasedCache.java

```diff
@@ -258,10 +258,11 @@ public synchronized void put(String key, Entry entry) {
             pruneIfNeeded();
             return;
```

Comment:
```suggestion
```

If move logic inside the catch block, then remove the return that is no longer needed.

Reply:
Done.